system-test/build.gradle

```diff
@@ -21,14 +21,14 @@ dependencies {
     compile 'com.github.docker-java:docker-java:1.3.0'
     compile 'com.mashape.unirest:unirest-java:1.4.5'
     compile 'com.jayway.awaitility:awaitility:1.6.3'
-    compile 'com.github.ContainerSolutions:mini-mesos:0.2.0'
+    compile 'com.github.ContainerSolutions:mini-mesos:c4ec773390329'
```

Comment:
My tests failed earlier when I used 0.2.0. To debug the problem I added extra logging to AbstractContainer.remove() https://github.com/ContainerSolutions/mini-mesos/blob/master/src/main/java/org/apache/mesos/mini/container/AbstractContainer.java#L105

When I upgraded and re-ran the test it succeeded. Probably because of the recent commits after 0.2.0